.eslintrc.json

```diff
@@ -93,6 +93,12 @@
           "react/sort-comp": 1,
           "react/wrap-multilines": 0,
           "react/jsx-filename-extension": 0,
-          "react/no-unescaped-entities":["error", { "forbid": [] }]
-        }
+          "react/no-unescaped-entities":["error", { "forbid": [] }],
```

Comment:
@olajide1234 @Dubby20 @leksyib @youngestdj @kleva-j what is the resolution about the concerns raised by @chingsley?

Reply:
@Dubby20 Kindly check this out.

Reply:
![image](https://user-images.githubusercontent.com/29398098/55561779-b1647e80-56ea-11e9-980a-8ea143cb9f1e.png)